app/forms/admin/new_user_form.rb

```diff
@@ -10,23 +10,18 @@ class NewUserForm
     validates :can_manage_others, presence: false, inclusion: [true, nil]
 
     def save
-      return unless valid?
+      return false unless valid?
 
       # TODO: look at getting boolean from form directly
       # seems to be issue with DFE form builder
       boolean_can_manage_others = can_manage_others ? true : false
 
-      begin
-        user = User.new(
-          email: email,
-          can_manage_others: boolean_can_manage_others
-        )
-
-        user.save
-      rescue ActiveRecord::RecordNotUnique
-        errors.add(:email, :uniqueness)
-        false
-      end
+      User.create!(email: email, can_manage_others: boolean_can_manage_others)
+      NotifyMailer.access_granted_email(email).deliver_now
+      true
+    rescue ActiveRecord::RecordNotUnique
+      errors.add(:email, :uniqueness)
```

Comment:
Why not use a uniqueness validation up front? 🤔 
https://guides.rubyonrails.org/active_record_validations.html#uniqueness

Reply:
Because we're using a database constraint which is more reliable when enforcing uniqueness. We could add an additional rails validation I suppose, but I'd want that to be alongside the database unique index.